tia-client/src/main/java/com/teamscale/tia.client/TiaAgent.java

```diff
@@ -59,9 +62,17 @@ public TestRun startTestRun() {
 	}
 
 	/**
-	 * Runs the TIA to determine which of the given available tests should be run and in which order.
+	 * Runs the TIA to determine which of the given available tests should be run and in which order. This method
+	 * considers all changes since the given baseline timestamp.
 	 *
-	 * @param baseline Considers all changes since this date.
+	 * @param availableTests A list of all available tests. This is used to determine which tests need to be run, e.g.
+	 *                       because they are completely new or changed since the last run. If you provide an empty
+	 *                       list, no tests will be selected. If you provide null, Teamscale will perform the selection
+	 *                       and prioritization based on the tests it currently knows about. In this case, it will not
```

Comment:
Instead of passing in null we probably need to create an overloaded version of `startTestRun` without `availableTests` that considers the different type of result returned by the agent. (PrioritizableTest)